.github/workflows/event-merge-to-queue.yml

```diff
@@ -16,40 +16,52 @@ concurrency:
 # TODO: Use RedisJSON's `${{ vars.DEFAULT_REDISJSON_REF }}` branch when testing on nightly
 
 jobs:
+  get-latest-redis-tag:
+    uses: ./.github/workflows/task-get-latest-tag.yml
+    with:
+      repo: redis/redis
+
   test-linux:
+    needs: get-latest-redis-tag
     uses: ./.github/workflows/flow-linux-platforms.yml
     secrets: inherit
     with:
-      redis-ref: unstable
+      redis-ref: ${{ needs.get-latest-redis-tag.outputs.tag }}
       rejson-branch: master
 
   test-macos:
+    needs: get-latest-redis-tag
     uses: ./.github/workflows/flow-macos.yml
     secrets: inherit
     with:
-      redis-ref: unstable
+      redis-ref: ${{ needs.get-latest-redis-tag.outputs.tag }}
       rejson-branch: master
 
   coverage:
+    needs: get-latest-redis-tag
     if: ${{ vars.ENABLE_CODE_COVERAGE != 'false'}}
-    uses: ./.github/workflows/flow-coverage.yml
+    uses: ./.github/workflows/task-test.yml
     secrets: inherit
     with:
+      coverage: true
+      get-redis: ${{ needs.get-latest-redis-tag.outputs.tag }}
       rejson-branch: master
+      test-config: '' # run all tests
 
   sanitize:
+    needs: get-latest-redis-tag
     secrets: inherit
     uses: ./.github/workflows/task-test.yml
     with:
-      get-redis: unstable
+      get-redis: ${{ needs.get-latest-redis-tag.outputs.tag }}
       rejson-branch: master
       test-config: '' # run all tests
       san: address
       env: ubuntu-latest
-      run_miri: true
```

Comment:
Why stop running miri ?

Reply:
Not stopping, only removing the explicit input.
We use `miri` if this is a sanitizer flow.